.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature Request
```

Comment:
Do we want to do this, or should we only encourage people to send in feature requests via the forums?

Reply:
I think we still want to allow people to request features here just to lower the barrier-to-entry for anyone who might not have a Forums account. We can always follow up in the GitHub Issues UI asking them to post on the forums if it's a larger feature request.